src/poetry/console/commands/check.py

```diff
@@ -19,6 +25,32 @@ def handle(self) -> int:
         poetry_file = Factory.locate(Path.cwd())
         config = PyProjectTOML(poetry_file).poetry_config
         check_result = Factory.validate(config, strict=True)
+
+        unrecognized = sorted(
+            set(config.get("classifiers", []))
```

Comment:
It's small but I'd rather only run `set(<value>)` and `config.get()` once.

Reply:
Good catch! Fixed in: https://github.com/python-poetry/poetry/pull/2881/commits/2eb0354983d49ca7a4d4aead37f3e2d9cc1e82d3